front/src/modules/ui/navbar/components/TaskNavMenuItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
we should not introduce new components but re-use the existing one (NavItem and allowing to pass a count).
We should also add Storybook tests to cover this NavItem configuration.